sources/packages/backend/apps/api/src/route-controllers/program-year/models/program-year.dto.ts

```diff
@@ -1,6 +1,6 @@
-export class ProgramYearAPIOutDTO {
+export class ProgramYearAndFormDetailsAPIOutDTO {
```

Comment:
This class does not seem to be used anywhere. Can you please verify it?

Reply:
Yes it is not used. Removed the DTO.